roads.mss

```diff
@@ -1,25 +1,25 @@
-/* ==================================================================
-                    ROAD & RAIL LINES
-/* ================================================================== */
+// ==================================================================
```

Comment:
Is it just for beauty or does `//` vs `/*` have a broader impact?

Reply:
/* is much more slower when you test optimisation code by commenting big section of the file :D

Reply:
Wow